app/helpers/statistics_helper.rb

```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```

Comment:
Well I can't explain but I can assure you that it works with and doesn't work without :/